docs/reference/sql/functions/aggs.asciidoc

```diff
@@ -113,6 +113,185 @@ Returns the total number of _distinct non-null_ values in input values.
 include-tagged::{sql-specs}/docs.csv-spec[aggCountDistinct]
 --------------------------------------------------
 
+[[sql-functions-aggs-first]]
+===== `FIRST/FIRST_VALUE`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+FIRST(field_name<1>)
+FIRST_VALUE(field_name<1>)
+
+FIRST(field_name<1>, ordering_field_name<2>)
+FIRST_VALUE(field_name<1>, ordering_field_name<2>)
+--------------------------------------------------
+
+*Input*:
+
+<1> target field for the aggregation
```

Comment:
"aggregation" refers to technical implementation in the background. My personal approach would be not to expose this, but try to explain what the field is used for from the end-user perspective. Again, personal preference.

Reply:
Not necessarily - for the aggregate function (aggregation is ES terminology, aggregate is SQL).